_data/sites.yml

```diff
@@ -957,6 +962,9 @@
   url: https://matthew.science/
   size: 276
   last_checked: 2021-12-19
+  
```

Comment:
Please remove the extra blank lines created

Reply:
hmm imma fix it rn

Reply:
this should fix it